docs/source/guides/publishing.rst

```diff
@@ -42,4 +44,4 @@ After you have set up the folder structure, head to `https://northstar.thunderst
 
 When uploading, it will verify your package structure and you can publish after it's successfully checked.
 
-To update a mod, change the version in ``mod.json`` and ``manifest.json``, and upload again. If the mod name is the same, it will update the previous version.
```

Comment:
No idea why this line is marked as changed. Line ending maybe?

Reply:
No clue, looks fine tho